lib/addons/aws-for-fluent-bit/index.ts

```diff
@@ -45,10 +51,12 @@ export class AwsForFluentBitAddOn extends HelmAddOn {
 
     deploy(clusterInfo: ClusterInfo): Promise<Construct> {
         const cluster = clusterInfo.cluster;
-
-        // Create the FluentBit namespace.
-        const namespace = this.options.namespace;
-        createNamespace(this.options.namespace!, cluster, true);
+        const namespace = this.options.namespace!;
+        
+        // Create namespace
+        if (this.options.createNamespace) {
+            createNamespace(namespace, cluster, true);
```

Comment:
Missing dependency between the namespace and service account, it will fail intermittently. 
Similar pattern [here](https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/lib/addons/ack/index.ts#L77).